robot.py

```diff
@@ -184,6 +187,11 @@ def teleopPeriodic(self):
         5: Hatch Panel grab (piston out). 5+wammy: Hatch Panel release (piston in). (5, z!=0)
         Start: Activate end game with Driver approval (8)
         '''
+      
+        #timer 
+
+        if self.matchtimer.timer.hasPeriodPassed(1):
```

Comment:
You should hide the timer interface from the robot.   Implement your own hasPeriodPassed(n) so the user can call it from the matchTimer.hasPeriodPassed(1)